### PR TITLE
Changed 5yr GW legend to match GW specifications

### PR DESCRIPTION
**Pull request by dpattermann-usgs**
Max is the smallest number, because numbers are being measured as distance below ground level (so the smallest number is the highest up).
Also, fixed fiveyrgwsum(data) so that it works.

### CODE REVIEW

**Comment by dpattermann-usgs** [comment]
@lindsaycarr @thongsav-usgs Think this should do it. If you think I should change it lower down rather than directly in the legend, say so.

**Comment by lindsayplatt** [comment]
I think directly in the legend is good

File: R/fiveyeargwsum-styles.R [hunk 1]
```diff
@@ -5,9 +5,9 @@ getFiveyearStyle <- function(data) {
     styles <- switch(names(data), 
                      stat = list(lines = list(x=x, y=y, type="o", col="black", pch=20, cex=0.5, lwd=0.8, legend.name=data[[1]]$legend.name)),
                      est_stat = list(lines = list(x=x, y=y, col="red", lty=2, legend.name=data[[1]]$legend.name)),
-                     max_iv = list(points = list(x=x, y=y, col="red", pch=8, cex=2, legend.name=paste("Max. Instantaneous Water Level:", y)),
+                     max_iv = list(points = list(x=x, y=y, col="red", pch=8, cex=2, legend.name=paste("Min. Instantaneous (Lowest) Water Level:", y)),
```
Comment (lindsayplatt) [inline R/fiveyeargwsum-styles.R]: @dpattermann-usgs could you have it say "Min. (Lowest) Instantaneous..." instead? Same for max - just move where the word in parentheses is?

Comment (dpattermann-usgs) [inline R/fiveyeargwsum-styles.R]: Ah! That's a good idea. Will do.

**Pull request merged**